Common/GltfExport.cs

```diff
@@ -0,0 +1,182 @@
+﻿using GaneshaDx.Resources.ContentDataTypes.Polygons;
+using System.Numerics;
+using System.Text.Json;
+using SharpGLTF.Geometry;
+using SharpGLTF.Materials;
+using SharpGLTF.Memory;
+using SharpGLTF.Schema2;
+using SharpGLTF.Geometry.VertexTypes;
+using System.IO;
+using GaneshaDx.Resources;
+using System.Collections.Generic;
+using GaneshaDx.Environment;
+using Microsoft.Xna.Framework;
+using Vector4 = System.Numerics.Vector4;
+using Vector3 = System.Numerics.Vector3;
+using Vector2 = System.Numerics.Vector2;
+using MTex2D = Microsoft.Xna.Framework.Graphics.Texture2D;
+
+namespace GaneshaDx.Common
+{
+  public static class GltfExport {
+
+		/**
+		* Exports the map model to the .gltf format.
```

Comment:
Please remove this Comment. The Comment philosophy is to only have comments to explain "Why", not "What". Here, we already know what this method does because of its name and its class.